boards/build_scripts/tock_kernel_layout.ld

```diff
@@ -99,24 +103,25 @@ SECTIONS
          * These are expected to just be empty on other platforms so they
          * shouldn't have any effect.
          */
-        KEEP(*(.riscv.start));
+        KEEP (*(.riscv.start));
         /* For RISC-V we need the `_start_trap` function to be 256 byte aligned,
          * and that function is at the start of the .riscv.trap section. If that
          * function does not exist (as for non-RISC-V platforms) then we do not
          * need any unusual alignment.
-         * The allignment is implementation specific, so we currently use 256 to
+         * The alignment is implementation specific, so we currently use 256 to
          * work with the lowRISC CPUs.
          */
         . = DEFINED(_start_trap) ? ALIGN(256) : ALIGN(1);
-        KEEP(*(.riscv.trap_vectored));
-        KEEP(*(.riscv.trap));
+        KEEP (*(.riscv.trap_vectored));
+        KEEP (*(.riscv.trap));
 
         /* x86 */
-        KEEP(*(.x86.start))
+        KEEP (*(.x86.start))
 
-        /* .text and .rodata hold most program code and immutable constants */
+        /* .text and .rodata hold most program code and immutable constants
```

Comment:
```suggestion
        /* .text and .rodata hold most program code and immutable constants */
```
?, Or I guess remove the `/*` on the line 122, I think of these as separate comments, though I see that they are describing the same directive.